cmd/build_worker_image.go

```diff
@@ -105,6 +105,7 @@ func (b *workerImageBuilder) build(ctx context.Context) error {
 		// including semver build metadata.
 		langTagComponent := lang + "-" + strings.TrimPrefix(b.version, "v")
 		b.tags = append(b.tags, omesVersion+"-"+langTagComponent)
+		b.tags = append(b.tags, lang+"-"+omesVersion)
```

Comment:
Also publish under the tag like `go-979932c` so CICD can automatically find the worker version for its Omes dependency.